packages/orderbook/README.md

```diff
@@ -0,0 +1,470 @@
+# Raindex SDK
+
+A TypeScript/JavaScript SDK for interacting with orderbook contracts, providing comprehensive functionality for order management, configuration parsing, and blockchain interactions.
+
+## Overview
+
+This SDK provides Rust-powered WebAssembly bindings for orderbook functionality, enabling developers to:
+
+- Parse and validate YAML configuration files
+- Generate order hashes and calldata
```

Comment:
is generating order hashes really key functionality?

Reply:
i'll rework this section